cmd/containerboot/tailscaled.go

```diff
@@ -173,11 +173,15 @@ func tailscaleSet(ctx context.Context, cfg *settings) error {
 func watchTailscaledConfigChanges(ctx context.Context, path string, lc *local.Client, errCh chan<- error) {
 	var (
 		tickChan          <-chan time.Time
+		eventChan         <-chan fsnotify.Event
+		errChan           <-chan error
 		tailscaledCfgDir  = filepath.Dir(path)
 		prevTailscaledCfg []byte
 	)
 	w, err := fsnotify.NewWatcher()
 	if err != nil {
```

Comment:
nit: Could optionally scope the `w` variable to this if statement to help avoid bugs from future edits as well.

```suggestion
	if w, err := fsnotify.NewWatcher(); err != nil {
```

Reply:
nice